lua/entities/gmod_tardis/modules/teleport/sh_tp_main.lua

```diff
@@ -369,9 +369,9 @@ else
                 if pos and self:GetData("demat-fast",false) then
                     if not IsValid(self) then return end
                     if (self:GetData("demat-fast",false))==true then
-                        sound.Play(ext.mat_damaged, pos)
+                        sound.Play(ext.mat_fast, pos)
                     else
-                        sound.Play(ext.mat, pos)
```

Comment:
Is it even possible to hit this code? We check for demat-fast on line 369 and then again on 371

Probably can just delete that 2nd if statement unless I'm missing something

Reply:
Looking further down it seems the 2nd if statement should instead be checking for `self:GetData("health-warning", false)` like it does on line 407, maybe it should be doing that. Do we have a fast mat sound for when it's damaged?

Reply:
> Looking further down it seems the 2nd if statement should instead be checking for `self:GetData("health-warning", false)` like it does on line 407, maybe it should be doing that. Do we have a fast mat sound for when it's damaged?

No but i can make one.

Reply:
Cool okay that'd be nice. I assume we need the fast mat sound to ensure it all syncs up nicely with the visuals when doing a fast demat and can't just use the regular mat sound?

Reply:
> Looking further down it seems the 2nd if statement should instead be checking for `self:GetData("health-warning", false)` like it does on line 407, maybe it should be doing that. Do we have a fast mat sound for when it's damaged?

It's not health warning. I tried to change the sound and it was the same as 1st in warning state

Reply:
> > Looking further down it seems the 2nd if statement should instead be checking for `self:GetData("health-warning", false)` like it does on line 407, maybe it should be doing that. Do we have a fast mat sound for when it's damaged?
> 
> It's not health warning. I tried to change the sound and it was the same as 1st in warning state

It seems like it's actually doesnt have any use

Reply:
Is that with that code change in place? Right now it seems useless but if you change the if condition it might start doing something

Reply:
> Is that with that code change in place? Right now it seems useless but if you change the if condition it might start doing something

Done